requirements/prod.in

```diff
@@ -25,7 +26,7 @@ salesforce-bulk
 sarge
 selenium
 simple-salesforce
-snowfakery
+snowfakery==2.0.dev0
```

Comment:
Need to make sure this gets updated to a final release of snowfakery before merging.